openupgrade_scripts/scripts/website/15.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,343 @@
+---Models in module 'website'---
+new model theme.ir.asset
+new model website.configurator.feature
+new model website.searchable.mixin [abstract]
+# NOTHING TO DO
+
+---Fields in module 'website'---
+website      / ir.asset                 / key (char)                    : NEW
+website      / ir.asset                 / theme_template_id (many2one)  : NEW relation: theme.ir.asset
+website      / ir.asset                 / website_id (many2one)         : NEW relation: website
+# NOTHING TO DO
+
+website      / res.company              / website_id (many2one)         : NEW relation: website, isfunction: function, stored
+# NOTHING TO DO
+
+website      / theme.ir.asset           / active (boolean)              : NEW hasdefault
+website      / theme.ir.asset           / bundle (char)                 : NEW required
+website      / theme.ir.asset           / copy_ids (one2many)           : NEW relation: ir.asset
+website      / theme.ir.asset           / directive (selection)         : NEW selection_keys: ['after', 'append', 'before', 'include', 'prepend', 'remove', 'replace'], hasdefault
+website      / theme.ir.asset           / key (char)                    : NEW
+website      / theme.ir.asset           / name (char)                   : NEW required
+website      / theme.ir.asset           / path (char)                   : NEW required
+website      / theme.ir.asset           / sequence (integer)            : NEW required, req_default: function, hasdefault
+website      / theme.ir.asset           / target (char)                 : NEW
+# NOTHING TO DO: new model
+
+website      / website                  / _order                        : _order is now 'sequence, id' ('id')
+website      / website                  / configurator_done (boolean)   : NEW
+website      / website                  / sequence (integer)            : NEW hasdefault
+# NOTHING TO DO
+
+website      / website                  / custom_code_footer (text)     : type is now 'html' ('text')
+website      / website                  / custom_code_head (text)       : type is now 'html' ('text')
+website      / website                  / robots_txt (text)             : type is now 'html' ('text')
+# DONE: pre-migation: convert custom_code_footer, custom_code_head, robots_txt field from text to html
```

Comment:
No transformation should be done here. They are still shown the same way as XML with widget=ace.